### PR TITLE
Add docs pages for the username recovery multi channel support in guides.

### PR DESCRIPTION
**Pull request by Malith-19**
## Purpose
- Introducing guide page for the username recovery under the `user-self-service/self-service-portal`
- Introducing guide page for the username recovery under the `Your identity server`
- Add docs for the non unique user support configuration.

## Related PR s
- https://github.com/wso2/docs-is/pull/4816

## Related Issues
- https://github.com/wso2/product-is/issues/21451
- https://github.com/wso2/product-is/issues/21180

### Self service portal
image

image

image

### Your identity server
image

image

### CODE REVIEW

File: en/identity-server/next/docs/guides/your-is/recover-username.md [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+
+1. Go to the sign-in page of the [My Account Portal]({{base_path}}/guides/your-is/is-self-service/#access-my-account-portal).
+
+    ![Recover your username]({{base_path}}/assets/img/guides/organization/self-service/customer/recover-your-username.png){: width="300" style="display: block; margin: 0; border: 0.3px solid lightgrey;"}
+
+2. Click **Forgot username?**.
+
+3. Enter the information for the account you’re trying to recover, then click **Next**.
+
+    ![Username recovery claiim]({{base_path}}/assets/img/guides/organization/self-service/customer/username-recovery-claim.png){: width="300" style="display: block; margin: 0; border: 0.3px solid lightgrey;"}
```
Comment (ashensw) [inline en/identity-server/next/docs/guides/your-is/recover-username.md:15]:
```suggestion
    ![Username recovery claim]({{base_path}}/assets/img/guides/organization/self-service/customer/username-recovery-claim.png){: width="300" style="display: block; margin: 0; border: 0.3px solid lightgrey;"}
```

File: en/identity-server/next/docs/guides/user-self-service/user-username-recovery.md [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+
+Users may recover a forgotten username from an application sign-in page by following the steps below.
+
+!!! note "Before you begin"
+
+    Administrators should [enable username recovery]({{base_path}}/guides/account-configurations/account-recovery/username-recovery/) for users of the organization.
+
+## Recover a forgotten username
+
+To reset the username, the user should follow these steps.
```
Comment (ashensw) [inline en/identity-server/next/docs/guides/user-self-service/user-username-recovery.md:11]: reset -> recover?

File: en/identity-server/next/docs/guides/your-is/recover-username.md [hunk 3]
```diff
@@ -0,0 +1,23 @@
+# Recover your username
+
+If you are the owner or an administrator of an WSO2 Identity Server organization, the following guide explains how you can recover a forgotten username.
```
Comment (ashensw) [inline en/identity-server/next/docs/guides/your-is/recover-username.md:3]:
```suggestion
If you are the owner or an administrator of a WSO2 Identity Server organization, the following guide explains how you can recover a forgotten username.
```

Comment (Malith-19) [reply en/identity-server/next/docs/guides/your-is/recover-username.md]: Fixed by https://github.com/wso2/docs-is/pull/4858/commits/7c797952be0d1b954991ed2889ed15992b0564b4

File: en/identity-server/next/docs/guides/user-self-service/user-username-recovery.md (hunk 2 above)
Comment (Malith-19) [reply en/identity-server/next/docs/guides/user-self-service/user-username-recovery.md]: Fixed by https://github.com/wso2/docs-is/pull/4858/commits/7c797952be0d1b954991ed2889ed15992b0564b4

File: en/identity-server/next/docs/guides/your-is/recover-username.md (hunk 1 above)
Comment (Malith-19) [reply en/identity-server/next/docs/guides/your-is/recover-username.md]: Fixed by https://github.com/wso2/docs-is/pull/4858/commits/7c797952be0d1b954991ed2889ed15992b0564b4

File: en/identity-server/next/docs/guides/user-self-service/user-username-recovery.md [hunk 4]
```diff
@@ -0,0 +1,40 @@
...
+
+1. Go to a login page via a business application or the [My Account portal]({{base_path}}/guides/user-self-service/customer-self-service-portal/).
+
+    ![Recover your password]({{base_path}}/assets/img/guides/organization/self-service/customer/recover-your-username.png){: width="300" style="display: block; margin: 0; border: 0.3px solid lightgrey;"}
+
+2. Click **Forgot username?**
+
+3. Enter the information for the account you’re trying to recover, then click **Next**.
+
+    ![Username recovery claiim]({{base_path}}/assets/img/guides/organization/self-service/customer/username-recovery-claim.png){: width="300" style="display: block; margin: 0; border: 0.3px solid lightgrey;"}
```
[the remark this answers is not in the document]
Comment (Malith-19) [reply en/identity-server/next/docs/guides/user-self-service/user-username-recovery.md]: Fixed by https://github.com/wso2/docs-is/pull/4858/commits/7c797952be0d1b954991ed2889ed15992b0564b4

**Review by ashensw — APPROVED** [review]

**Pull request merged**